runtime/kusama/src/lib.rs

```diff
@@ -1513,10 +1513,31 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPallets,
+	SetStakingLimits,
 >;
 /// The payload being signed in the transactions.
 pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
+pub struct SetStakingLimits;
+impl frame_support::traits::OnRuntimeUpgrade for SetStakingLimits {
+	fn on_runtime_upgrade() -> Weight {
+		// This will be the threshold needed henceforth to become a nominator. All nominators will
+		// less than this amount bonded are at the risk of being chilled by another reporter.
+		let min_nominator_bond = UNITS / 10;
+		// The absolute maximum number of nominators. This number is set rather conservatively, and
+		// is expected to increase soon after this runtime upgrade via another governance proposal.
+		// The current Polkadot state has more than 30_000 nominators, therefore no other nominator
```

Comment:
There are currently around 22418 _active_ nominators on Polkadot, I wonder from where 30 000 number comes from?

Reply:
Its not about active nominators, its about number of nominators in the underlying `Nominators` storage map.

Reply:
> Its not about active nominators, its about number of nominators in the underlying `Nominators` storage map.

I see, thank you! So new nominators would not be able to start staking, or those who have higher stake would be stored, and this will be recalculated for each era?

Reply:
I would assume that only 20 000 with the highest stake will be counted

Reply:
This limit would make it so that only 20_000 accounts could exist in that nominators map. Once we hit that limit, other nominators would not be able to register.